src/app/components/info-drawer/comments-tab/comments-tab.component.ts

```diff
@@ -44,10 +44,15 @@ export class CommentsTabComponent {
   constructor(private permission: NodePermissionService) {}
 
   get canUpdateNode(): boolean {
-    if (this.node && this.node.isFile && !isLocked({ entry: this.node })) {
-      return this.permission.check(this.node, ['update']);
+    if (!this.node) {
+      return false;
     }
 
-    return false;
+    if (
+      this.node.isFolder ||
+      (this.node.isFile && !isLocked({ entry: this.node }))
+    ) {
+      return this.permission.check(this.node, ['update']);
+    }
```

Comment:
There's no "else" branch here, so the function will return "undefined"

Reply:
ups